### PR TITLE
issue 2340: Mixer volume percentage labels are off by a factor of 100

### PR DESCRIPTION
**Pull request by StCyr**
Simple fix for issue 2340

It seems to do the trick

### CODE REVIEW

**Comment by StCyr** [comment]
Corresponding issue: https://github.com/LMMS/lmms/issues/2340

File: src/gui/editors/AutomationEditor.cpp [hunk 1]
```diff
@@ -1044,7 +1044,7 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 			for( int i = 0; i < 2; ++i )
 			{
 				const QString & label = m_pattern->firstObject()
-						->displayValue( level[i] );
+						->displayValue( level[i] /100 );
```
Comment (Umcaruje) [inline src/gui/editors/AutomationEditor.cpp]: This should also use a space after the slash, like the other lines, for consistency: `/ 100`

**Comment by Umcaruje** [comment]
@StCyr Nice PR! it fixes the values in the context menus:
screenshot from 2016-02-26 17 55 26

Though I noticed some things in the tooltips that could also get fixed:

The volume tooltip looks ok, but it has a space before the `%` sign:
screenshot from 2016-02-26 17 50 41

It should look like `Volume: 109.4%` (without the space).

The send knob tooltip displays it as a decimal:
screenshot from 2016-02-26 17 50 15

Also it should have some descriptive text. Ideally it should look like `Send amount: 85.3%`

**Comment by Umcaruje** [comment]
Oh and in the automation editor the values are still decimal:
screenshot from 2016-02-26 18 01 28

**Comment by grejppi** [comment]
@StCyr @Umcaruje Meanwhile everywhere else...

100scale

**Comment by Umcaruje** [comment]
@grejppi ouch. Forgot to check that.

**Comment by StCyr** [comment]
you're right @grejppi; I didn't check that, sorry.

So, the problem is more difficult to handle than what I previously thought.

After some debugging, it appears that Faders store their value in a range from 0 to 2, while Knobs use a range from 0 to 200. 

I guess, solving the problem will require unifying these behaviors, which will probably be quite invasive.

For the record, here are some debugging outputs I could get:

```c++
Knob::getValue() value=-0,116667
AutomatableModel::setValue value=158,133347
Knob::getValue() value=-0,116667
AutomatableModel::setValue value=158,333344
Knob::getValue() value=-0,116667
AutomatableModel::setValue value=158,533340
Knob::getValue() value=-0,116667
AutomatableModel::setValue value=158,733337
Knob::getValue() value=0,000000
Knob::getValue() value=-0,116667
AutomatableModel::setValue value=158,933334
...
Fader::mouseMoveEvent value=1,442256
AutomatableModel::setValue value=1,442256
Fader::mouseMoveEvent value=1,465512
AutomatableModel::setValue value=1,465512
Fader::mouseMoveEvent value=1,488768
AutomatableModel::setValue value=1,488768
Fader::mouseMoveEvent value=1,512023
AutomatableModel::setValue value=1,512023
Fader::mouseMoveEvent value=1,535279
AutomatableModel::setValue value=1,535279
```

**Comment by tresf** [comment]
@diizy did some work with this back in the day.... https://github.com/LMMS/lmms/compare/models.  Not sure how much of it is relevant here.

**Comment by Umcaruje** [comment]
I'm closing this PR out to declutter the tracker, because this issue requires a lot more work. @StCyr if/when you come up with a solution to the problem, you can either open up a new PR or just push again to this one, and we'll reopen it.

**Pull request closed without merging**